index.html

```diff
@@ -1,278 +1,327 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mohammed's portoflio homepage</title>
+  </head>
```

Comment:
Kindly change the contrast to be at least `4.5` of the `My logo`, `Let's connect`, `Get my resume` and `Get it touch` , because they failed in the contract check
![screencapture-aditus-io-button-contrast-checker-raw-githack-com-2022-02-23-at-11-14-24-413-2022-02-23-13_15_12](https://user-images.githubusercontent.com/24703171/155309379-717f04f5-fa32-4e6e-ac5c-12d346527392.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)